### DISCUSSION TITLE
what about Palestinian flag?!

### DISCUSSION THREAD

**Discussion by MatejMagat305** [xoureldeen/Vectras-VM-Android#80] [General]
Look, ach, I don't want to open politic topic, I don't want to be arogant, but I rather be stupid at judment day as be full sins of hated someone, I am sorry, but this is so important, that I must, so:

1. Really Palestian flag at this ap? Why? Do you support Palestinians? If yes, do you have historically right solution?
2. Why not Izrael flag too? Because even in Izrael are palestinians too... (and if you want 2 states, you should show even second state, am I wrong?)
3. I don't like kill people! But can you really (honnestly) tell me why that is main goal of Hamas? Why it has goal to kill all Jews? At Quran, at hadiths, are Jews mainly enemy, but I ask to fair and unfair ...
4. can I please you look at this video and can you tell me, where they are wrong (in history): https://m.youtube.com/watch?v=BJiX9_spvak?
5. if you are muslim, can you tell me, where they are wrong at describe relavation of Jews and Muslims https://m.youtube.com/watch?v=w3NX4VnNuV8?
6. can you tell me, Palestinians are Arabs, who got big finance support from EUropian union, why they put majority of them to warpour?
7. how I can be fair? I want be fair..., Izrael is only state on entire world, Islam has 45+ states, where should Jews go? (When I see from land to sea, Palestine will be free) Palestinians can run away to Egypt, Syria ..., but where can Jews go? Can Hamas openly declared genocide and they are good, but Jews are bad even (even if Jews would be equal as Hamas, I would be unfair)

I am sorry, but I am stupid and I ask and I want antwer..., again I know even Jews make errors, but what do you want? Gaza in history reject state I don't have solution, do you know what would make Muslims less angry to Jews? I try to learn Quran and I have no idea...

EDIT: 
I want to clarify that (after years) I do not support the politics of the current Israeli government either. I have also watched multiple videos showing the harsh, unacceptable actions and wrongdoings committed from the Israeli side—the videos I linked above were simply provided as examples from the "other side" to balance the common narrative.

In my view, both nations should ideally be under an international protectorate and forced into peace, though who could act as a truly neutral guardian today is a open question. Ultimately, from my Christian perspective, I believe true and lasting peace will never be achieved until both sides accept Jesus as the Messiah.

**Comment by ahmedbarakat2007** [comment]
you see what is happening in palastine and how many people dies there i don't need to explain

**Reply by MatejMagat305** [reply]
Ok, how much Hamas want kill? ALL Jews, what would you doing when you would live run away, no? Why Hamas forbit this? Why your brothers gratulation Hitler for 2 milions killed? Why you do not antwer me questions Historicaly?  I want peace, but NOT to cancel Izrael and NOT give Hamas chance to rise again!!

How many mouqse is in Izrael? (many) How many citizen Arab lives in Izrael? 2.2 milions from 160 000 a 70 years ago, in Gaza 14 000 to 0, from around countries must leave 850 000+ Jews, because otherway they would be killed, in Gaza was 70 000 citizen, now 2.2 milions, so if I be fair citizen of Gaza must leave or what?

yes it would be better open coridors, but Egypt do nothing ..., Jordan Palestinian still not conciderate to own people

OK, but state of Hamas will in permanent war with Izrael state the result will be more worst than now! So what is your solution, kill ALL Jews? Or else?

**Reply by MatejMagat305** [reply]
Well, number are horible, but small analogy, if 50 men are prepare kill you and you call police, they try arrest this people and at the arrest kill 5 of them will you guilty?

**Comment by xoureldeen** [comment]
+ THE FIRST DEVELOPER OF THIS APP IS PALESTINIAN AND KILLED DUE THE WAR

**Reply by xoureldeen** [reply]
ALSO HIS 8~12 AGE BROTHERS, ALSO HE IS 16 
BRUH

**Reply by MatejMagat305** [reply]
thank you, but still I wait for Historical antwer ..., I cry for people, but if you do not cry for hated shout on al-aqsa, where on youtube shout death all Jews, then what wait from Jews, do you told me we will not doing, what is your granted of these? Or Quran say you can lie and kill Jews, but Jews can not be equal as you?

**Reply by xoureldeen** [reply]
بص انا هقولك كلمه واحده, كىىىمك

**Reply by xoureldeen** [reply]
![image](https://github.com/epicstudios856/Vectras-VM-Android/assets/79816069/6acb450f-3517-4213-aa1e-202459332647)

**Reply by MatejMagat305** [reply]
Well, number are horible, but small analogy, if 50 men are prepare kill you and you call police, they try arrest this people and at the arrest kill 5 of them will you guilty?

Hamas break sign of peace from 2005 when 10 000 Jews leave Gaza until now Hamas shot to Izrael by rocket, many years are in youtube videa from al-aqsa m. Claiming we go to kill all Jews ...

When Izrael was created Arab was attack next day and reject peace in 1948, then 1967 again, they were lost, therefore lost peace of land, that is totaly normal ...

From Arab country was kick out 800 000+ Jews, because etnic cleaning, now Muslim attack Jews even in western countries, Izrael tolerate tousen of moqs (pray building of Muslim), in Gaza is not only Sinagoges, 

I never claim that Jews are inocent (everybody has errors), but if you judge they, you will be judge by Yahve for this, yes I am verry stupid christian (i believe the only way to God is accept Jesus as saviour for sins), but I do not want you to be judge for unfaireness

**Reply by xoureldeen** [reply]
dont use the app anymore

we are Terrorists

**Reply by MatejMagat305** [reply]
I never claim you, or all muslims are terorist, I claim that, there are understandable facts, why Izrael is so hard, and you should know facts, European union have supported Gaza and I asumme that it will support Gaza again, I will agree with that, but I would like to see build infractructure and NOT buying warpour ...

but I disagree to cancel state Izrael and/or give Hamas freedom to his agenda ...

It is also paradox that even in Jordan or Libadon Palestinians are NOT as regular citizens they are still refugee, even Moslim countries do not want them ..., then I ask why when Moslim world reject Palestinians, Izrael should be ok with Hamas ...

And historic ownership?! Then Arab should return part of Mecca to Jews, it is real, NO, so it is NOT real to have Palestine from 1948, Gaza was and probably will autonome area which will have own govermant, but they must NOT allow attack own solder to Izrael, because this is war

So I only want that you know about this situation is complication and true is this lie, hate, enemity started false prophet Mohamed, who hated Jews and was not good model of humanity, I am not happy to Jews kill that numbers, but Hamas still fight agains them even know they not allow to midlier duration ...

**Comment by adk23333** [comment]
> + THE FIRST DEVELOPER OF THIS APP IS PALESTINIAN AND KILLED DUE THE WAR

Most Chinese people understand Palestine, as we have experienced similar things, but we are now strong. In this era, we have committed to not directly interfering in regional conflicts, but the truth is fully protected on the Chinese internet.
